tests/integration/targets/test_azure_manage_postgresql/tasks/create_and_delete_postgresql.yml

```diff
@@ -39,7 +38,7 @@
           - _postgresql.servers[0].sku.name == azure_manage_postgresql_postgresql_sku.name
           - _postgresql.servers[0].storage_mb == azure_manage_postgresql_postgresql_storage_mb
           - _postgresql.servers[0].admin_username == azure_manage_postgresql_postgresql_admin_username
-          - _postgresql.servers[0].version == azure_manage_postgresql_postgresql_version
+          - _postgresql.servers[0].version == "11"
```

Comment:
why not azure_manage_postgresql_postgresql_version?

Reply:
I removed azure_manage_postgresql_postgresql_version from the test and we are using the role's default value

Reply:
ok